docs/deploy/agent-engine/deploy.md

```diff
@@ -0,0 +1,188 @@
+
+2. **Create a Google Cloud project**
+    * If you already have an existing Google Cloud project, you can use that and do not create a new one.
+    * If you want to create a new Google Cloud project, you can create a new one on the [Create Project](https://console.cloud.google.com/projectcreate) page.
+
+3. **Get your Google Cloud project id**
+    * You need your Google Cloud project id, which you can find on your GCP homepage. Make sure to note the project id (alphanumeric with hyphens), _not_ the project number (numeric).
+
+    <img src="/adk-docs/assets/project-id.png" alt="Google Cloud Project ID">
+
+4. **Enable Vertex AI in your project**
+    * To use Agent Engine, you need to [enable the Vertex AI API](https://console.cloud.google.com/apis/library/aiplatform.googleapis.com). Click on the "Enable" button to enable the API. Once enabled, it
+    should say "API Enabled".
+
+5. **Enable Cloud Resource Manager API in your project**
+    * To use Agent Engine, you need to [enable the Cloud Resource Manager API](https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview). Click on the "Enable" button to enable the API. Once enabled, it should say "API Enabled".
+
+6. **Create a Google Cloud Storage (GCS) Bucket**:
+    * Agent Engine requires a GCS bucket to stage your agent's code and dependencies for deployment. If already have one, we recommend you create a new one.
+    * Create a GCS bucket by following the instructions [here](https://cloud.google.com/storage/docs/creating-buckets). We recommend starting with the default settings when creating your first bucket.
+    * Once you have created a storage bucket, you should be able to see it on the [Cloud Storage Buckets page](https://console.cloud.google.com/storage/browser).
+    * You need the GCS bucket path to set as your staging bucket. For example, if your GCS bucket name is "my-bucket", then your bucket path should be "gs://my-bucket".
+
+## Set up your coding environment {#prerequisites-coding-env}
+
+Now that you prepared your Google Cloud project, you can return to your coding
+environment. These steps require access to a terminal within your coding
+environment to run command line instuctions.
+
+### Authenticate your coding environment with Google Cloud
+
+*   You need to authenticate your coding environment so that you and your
+    code can interact with Google Cloud. To do so, you need the gcloud CLI.
+    If you have never used the gcloud CLI, you need to first
+    [download and install it](https://docs.cloud.google.com/sdk/docs/install-sdk)
+    before continuing with the steps below:
+
+*   Run the following command in your terminal to access your Google Cloud
+    project as a user:
+
+    ```shell
+    gcloud auth login
+    ```
+
+    You should now see "You are now authenticated with the gcloud CLI!".
```

Comment:
Suggest adding "After authenticating, you should now see..." to indicate there are immediate steps to complete.